src/components/payment-section/components/payment-component/PaymentComponent.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { Sheet, Typography } from "@mui/joy";
+import styles from "./PaymentComponent.module.css";
+
+export function PaymentComponent({ label, src }) {
+  const [selectedValue, setSelectedValue] = useState("");
+
+  const handleChange = (event) => {
+    setSelectedValue(event.target.value);
+  };
+
+  return (
+    <Sheet
+      component="label"
+      key={label}
+      className={styles.paymentMethodContainer}
+    >
+      <input
+        type="radio"
+        name="radio"
+        value={label}
+        onChange={handleChange}
+        className={styles.input}
+      ></input>
+      <img alt={`${label}`} src={{ src }} className={styles.paymentMethodImg} />
```

Comment:
- why do a string template literal when you can just pass the prop like this : ```alt={label}``
- the src attribute won't work because you're passing an object with src prop instead of passing the prop directly like this : ```src={src}```